src/Microsoft.DocAsCode.Common/FileAbstractLayer/ManifestFileHelper.cs

```diff
@@ -59,5 +59,41 @@ public static bool AddFile(this Manifest manifest, string sourceFilePath, string
             }
             return false;
         }
+
+        public static void RemoveLinks(this Manifest manifest, string manifestFolder)
```

Comment:
RealizeLinkedFiles?

---
In reply to: [95947362](https://github.com/dotnet/docfx/pull/1172#discussion_r95947362) [](ancestors = 95947362)